### PR TITLE
[rector] [dead-code] Remove few ifs that are always true - part #2

### PR DESCRIPTION
**Pull request by TomasVotruba**
Younger brother of https://github.com/mautic/mautic/pull/12693

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Ready for review :heavy_check_mark:

**Comment by TomasVotruba** [comment]
Rebased :heavy_check_mark: 

---


This is good to go now :+1:  Basically 2nd part of previous just merged PR :)

File: app/bundles/FormBundle/Form/Type/FieldType.php [hunk 1]
```diff
@@ -301,39 +299,35 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             );
         }
 
-        if ($addInputAttributes) {
```
Comment (TomasVotruba) [inline app/bundles/FormBundle/Form/Type/FieldType.php]: I've just noticed the magic `$$f` naming above. That's why this value can be false/true, will update the PR.

**Review by escopecz — APPROVED** [review]
The code changes make sense. I dug through the code if the removed if statements won't cause any issues or performance degradation but it all looks like it's handled in lower layers.

Adding the return types to public methods is a BC break but since these changes would go to a major release, we can get it merged. We were forced to add types by our dependencies in multiple places anyway so a couple more shouldn't cause any surprises.

Thank you Tomas! 👍

**Pull request merged**